src/scripts/check_distance.py

```
@@ -74,17 +76,19 @@ async def check_distance():
         print(
-            f"{'Rank':<5} | {'Distance':<10} | {'Status':<10} | {'Law Info':<25} | {'Content Preview'}"
+            f"{'Rank':<5} | {'ID':<6} | {'Distance':<10} | {'Status':<10} | {'Law Info':<25} | {'Content Preview'}"
         )
         print("-" * 100)
 
         for i, row in enumerate(rows):
-            law_title = row[0]
-            article_no = row[1]
+            law_id = row[0]
+            law_type = row[1]
+            section_title = row[2] or ""
```

Ruff (0.15.2)

[error] Local variable `section_title` is assigned to but never used

Remove assignment to unused variable `section_title`

(F841)